File: luaMap/priem.lua

pi = 3.14159

print(pi)
print("Hello world")

function isPriem(num)
    for i = 2,(num-1) do
        if (num%i==0) then 
            return false
        end
    end
    return true
end


for a = 1,1000 do
    if isPriem(a) then
        print(a)
    end
end

File: priem.js
function isPriem(num) {
  for (let getal = 2; getal < num; getal++) {
    if (num % getal == 0) {
      return false;
    }
  }
  return true;
}

function priemLoop(params) {
  const priemGetallen = [];
  for (let i = 2; i < params; i++) {
    if (isPriem(i)) {
      priemGetallen.push(i);
    }
  }
  return priemGetallen;
}

const pr = priemLoop(4000);
console.log(pr);
